lib/cluster.js

```diff
@@ -104,11 +104,11 @@ function RoundRobinHandle(key, address, port, addressType, backlog, fd) {
   this.server = net.createServer(assert.fail);
 
   if (fd >= 0)
-    this.server.listen({ fd: fd });
+    this.server.listen({ fd: fd }, backlog);
   else if (port >= 0)
-    this.server.listen(port, address);
+    this.server.listen(port, address, backlog);
   else
-    this.server.listen(address);  // UNIX socket path.
+    this.server.listen(address, backlog);  // UNIX socket path.
```

Comment:
Do all of the variations of `listen()` accept a `backlog` parameter? It doesn't look like it's documented that way.